interview-questions.md

```diff
@@ -5,54 +5,55 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.  
 
-1. What are props in React?
+1. What are props in React? 
 
-  Your answer:
+  Your answer: Props are communication arroyos that allow information to flow downstream (one direction) from parent to child components. 
 
-  Researched answer:
+  Researched answer: Props, short for properties, are used to pass data between components in REACT. Data is assigned from the parent componenet to the child component(s) using the 'prop' attribue; using dot notation {props.name} to access and render.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: DOM is always watching and is always listening. BAsically DOM responds to user interaction. It could be a text type message or the click of a button, Regardless, you can't get anything passed DOM.
 
-  Researched answer:
+  Researched answer: DOM events are how HTML documents refister diffrent event handler elements when using Javasc. These events are paired with functions and will not execute before the function in imnvoked. 
```

Comment:
Good work. Lots of things can be a dom event. Scroll down on the NYTimes page and it will create a modal (a screen on the screen) that asks you to sign up for the NYTimes.

Reply:
NIce. And that was supposed to be register not refister. I can't spell.